alphastats/gui/pages/05_LLM.py

```diff
@@ -162,6 +192,11 @@ def llm_config():
 def llm_chat(llm_integration: LLMIntegration, show_all: bool = False):
     """The chat interface for the LLM analysis."""
 
+    # TODO dump to file -> static file name, plus button to do so
+    # how to deal with binaries? base64 encode?
+    # "import chat" functionality?
```

Comment:
I think that would be overkill. If we have a txt export, one could always sopy that into a prompt oneself and tell the LLM that this was the previous session.